src/main/java/com/cryptomorin/xseries/XItemStack.java

```diff
@@ -404,7 +467,7 @@ public static ItemStack deserialize(@Nonnull ConfigurationSection config) {
                     builder.trail(fw.getBoolean("trail"));
                     builder.with(Enums.getIfPresent(FireworkEffect.Type.class, fw.getString("type").toUpperCase(Locale.ENGLISH)).or(FireworkEffect.Type.STAR));
 
-                    List<String> fwColors = fw.getStringList("colors");
+                    List<String> fwColors = fw.getStringList("base-colors");
```

Comment:
Why?

Reply:
https://github.com/CryptoMorin/XSeries/blob/7e8144aa7788641e20f02b8ae5e34e88b1abe88c/src/main/java/com/cryptomorin/xseries/XItemStack.java#L223

Reply:
the colors section doesn't even exist.

Reply:
I see. I think it'd be better to change `base-colors` to `colors` instead? I'm trying to reuse simple names across multiple meta types for the sake of simplicity.

Reply:
The thing is that are two types of colors **base-colors** and **fade-colors**, if we change it then we couldn't differentiate them.

Reply:
I guess you're right. How about moving them to their own section like you did with the other meta?
```yml
colors:
  base: []
  fade: []
```

Reply:
Sure that would be nice.